src/gui/torrentcreatordialog.cpp

```diff
@@ -170,7 +183,11 @@ void TorrentCreatorDialog::dragEnterEvent(QDragEnterEvent *event)
 // Main function that create a .torrent file
 void TorrentCreatorDialog::onCreateButtonClicked()
 {
-    const auto inputPath = Utils::Fs::toCanonicalPath(Path(m_ui->textInputPath->text().trimmed()));
+    auto inputPath = Path(m_ui->textInputPath->text().trimmed());
+#ifdef Q_OS_WIN
+    // We don't want .lnk to be the root of the created torrent
```

Comment:
This comment is misleading as it isn't what happened in my testing. It will fail with a "path not exist" error returned from libtorrent/system if `toCanonicalPath()` isn't used.

Reply:
Does that apply to both cases either the shortcut itself or the subfolder of its target is used as torrent root?

Reply:
```suggestion
    // Resolve the path in case it contains a shortcut (otherwise, the following usages will consider it invalid)
```